Stratis.Bitcoin.Tests/BlockStore/LoopTests/BlockStoreLoopStepBaseTest.cs

```diff
@@ -89,42 +72,109 @@ internal Block CreateBlock(int blockNumber)
 
             return block;
         }
+    }
+
+    internal class FluentBlockStoreLoop : IDisposable
```

Comment:
Why the name Fluent? I remember we use to call it a TestContext.
Not suggesting to change just wondering